composables/transaction/transactionOffer.ts

```diff
@@ -13,7 +13,7 @@ export const getOfferCollectionId = (prefix: Prefix) => {
   }
 }
 
-const OFFER_MINT_PRICE = '500000000'
+export const OFFER_MINT_PRICE = '500000000'
```

Comment:
can we do `5e8`, so this becomes a number by default and we can make it a string where needed

Reply:
updated